app/components/appointmentCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { View, Text, StyleSheet, Pressable, Image } from "react-native";
+import { Button } from "react-native";
+import Colors from "../utils/Colors";
+
+const AppointmentCard = () => {
+    return (
+        <View style={styles.container}>
+                <Image style={styles.image} source={require("../../assets/appointmentImage.jpg")}/>
+            <View style={styles.card}>
+                <View style={styles.timer}>
+                    <Image style={styles.clockImage} source={require("../../assets/clock.png")}/>
+                    <Text style={styles.textTimer1}>Startet: </Text>
+                    <Text style={styles.textTimer2}>In 4 Tagen</Text>
```

Comment:
Could you add information that which would normally be dynamic, like the ``4`` in "4 Tage" and the title ``Grundlagen der Mathematik`` as props so that they get passed from the mother components.